app/build.gradle

```diff
@@ -28,6 +28,8 @@ dependencies {
     implementation 'org.slf4j:slf4j-simple:2.0.5'
     implementation 'org.reflections:reflections:0.10.2'
     implementation 'javax.inject:javax.inject:1'
+    implementation 'org.slf4j:slf4j-api:2.0.3'
+    implementation 'org.slf4j:slf4j-simple:2.0.3' // Can be replaced with logBack, log4j, log4j2, Java Utility Logger
```

Comment:
We already have this dependency a few lines above.

Reply:
Good catch